scrapyd_k8s/config.py

```diff
@@ -21,13 +21,27 @@ def launcher_cls(self):
         pkg, cls = repo.rsplit('.', 1)
         return getattr(import_module(pkg), cls)
 
+    def joblogs(self):
+        if self._config.has_section('joblogs'):
+            return self._config['joblogs']
+        else:
+            return {}
+
+    def joblogs_provider_args(self, provider):
```

Comment:
what about `joblogs_storage` for the function name?

Reply:
Ok with the name if you think it depicts the intention of the function better